generated/api/v1/policy_service.swagger.json

```diff
@@ -649,64 +649,75 @@
+          "description": "List of categories that this policy falls under.  Category names must already exist in Central."
         },
         "lifecycleStages": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/storageLifecycleStage"
-          }
+          },
+          "description": "Describes which policy lifecylce stages this policy applies to.  Choices are DEPLOY, BUILD, and RUNTIME."
         },
         "eventSource": {
-          "$ref": "#/definitions/storageEventSource"
+          "$ref": "#/definitions/storageEventSource",
+          "title": "Describes which events should trigger execution of this policy"
         },
         "exclusions": {
           "type": "array",
           "items": {
             "type": "object",
             "$ref": "#/definitions/storageExclusion"
-          }
+          },
+          "description": "Define deployments or images that should be excluded from this policy."
         },
         "scope": {
           "type": "array",
           "items": {
             "type": "object",
             "$ref": "#/definitions/storageScope"
-          }
+          },
+          "description": "Defines clusters, namespaces, and deployments that should be included in this policy.  No scopes defined includes everything."
         },
         "severity": {
-          "$ref": "#/definitions/storageSeverity"
+          "$ref": "#/definitions/storageSeverity",
+          "description": "Defines how severe a violation from this policy is.  Possible values are UNSET_SEVERITY, LOW_SEVERITY, MEDIUM_SEVERITY, HIGH_SEVERITY, and CRITICAL_SEVERITY."
         },
         "enforcementActions": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/storageEnforcementAction"
           },
-          "description": "FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.\nFAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.\nFAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates."
+          "description": "FAIL_DEPLOYMENT_CREATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object creates/updates.\nFAIL_KUBE_REQUEST_ENFORCEMENT takes effect only if admission control webhook is enabled to listen on exec and port-forward events.\nFAIL_DEPLOYMENT_UPDATE_ENFORCEMENT takes effect only if admission control webhook is configured to enforce on object updates.\nLists the enforcement actions to take when a violation from this policy is identified.  Possible value are UNSET_ENFORCEMENT, SCALE_TO_ZERO_ENFORCEMENT, UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT, KILL_POD_ENFORCEMENT, FAIL_BUILD_ENFORCEMENT, FAIL_KUBE_REQUEST_ENFORCEMENT, FAIL_DEPLOYMENT_CREATE_ENFORCEMENT, and. FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT."
```

Comment:
"Possible value are" - should be "possible values are" ?

extra period after "and" in last sentence